css/index.css

```diff
@@ -19,7 +21,17 @@ body {
   margin: 0;
   font-family: -apple-system, system-ui, sans-serif;
   color: var(--darkgray);
-  background-color: var(--white);
+  background-color: var(--light-blue);
```

Comment:
Looks great to me. I have a hard time choosing colors. @cmneal26 do you like this color choice?

Reply:
@twalker this isn't about your style choices. Just an honest question. Why no plant imagery?

Reply:
Do you not like plants, or is this about you showing it as a portfolio item to potential technical writers?